pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -94,22 +94,30 @@ func NewManager(
 func (m *managerImpl) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAdmitResult {
 	m.RLock()
 	defer m.RUnlock()
+	defer glog.Infof("eviction manager: end Admit: %v", attrs.Pod.Name)
+	glog.Infof("eviction manager: start Admit: %v", attrs.Pod.Name)
```

Comment:
I don't want any of these logs right now, it was purely for debugging right now locally to isolate that admit was not the problem.